tests/testthat/test-renderer2-interactive-legends.R

```diff
@@ -214,7 +214,7 @@ test_that('aes(color=vs.fac) is OK"', {
   clickID("plot_p_vs_fac_variable_0")
   expect_equal(length(get_circles("p")), 14)
   clickID("plot_p_vs_fac_variable_0")
-  expect_equal(length(get_circles("p")), 32)  
+  expect_equal(length(get_circles("p")), 32)
```

Comment:
@lazycipher please try to avoid committing changes like this line, where nothing substantial has changed (only whitespace) because such changes make code review more difficult.

Reply:
I might've missed that! I'll tak care of that from now on!